src/main/fc/init.c

```diff
@@ -1021,7 +1021,7 @@ void init(void)
 #endif
 
     // On H7/G4 allocate SPI DMA streams after motor timers as SPI DMA allocate will always be possible
-#if defined(STM32H7) || defined(STM32G4)
+#if defined(STM32H7) || defined(STM32G4) || defined(AT32F435)
```

Comment:
Should the comment be updated?
Does this not need SPI DMA streams allocated before motor timers like F4 and F7?

Reply:
I’ll update the comment. The F4/F7 are allocated before the motors to ensure the SPI DMA takes priority, but the G4/H7 have very flexible DMA channel allocation so can be done after, and the AT32 is the same.

Reply:
Done